src/vs/workbench/browser/parts/editor/editorPart.ts

```diff
@@ -1046,6 +1046,10 @@ export class EditorPart extends Part implements IEditorGroupsService, IEditorGro
 			type: Parts.EDITOR_PART
 		};
 	}
+
+	createDropTargets(container: HTMLElement, delegate: EditorDropTargetDelegate): IDisposable {
```

Comment:
@mjbvz I use folding in that file and would prefer if this new method could move up below `private assertGroupView`